setuptools/package_index.py

```diff
@@ -1014,7 +1004,7 @@ def __iter__(self):
         yield self.password
 
     def __str__(self):
-        return '%(username)s:%(password)s' % vars(self)
+        return '{username}:{password}'.format(**vars(self))
```

Comment:
Not an automated change, but this can be cleaned up further. Statically checkable too.

```suggestion
        return f'{self.username}:{self.password}'
```